I tried the instructions but mujoco failed to compile (Arch Linux):

>>> referenced by /usr/lib/gcc/x86_64-pc-linux-gnu/14.2.1/../../../../lib/Scrt1.o:(_start)
collect2: error: ld returned 1 exit status
make[2]: *** [simulate/CMakeFiles/simulate.dir/build.make:104: bin/simulate] Error 1
make[1]: *** [CMakeFiles/Makefile2:6886: simulate/CMakeFiles/simulate.dir/all] Error 2
make: *** [Makefile:146: all] Error 2

--- a/README.md
+++ b/README.md
@@ -21,6 +21,7 @@ tar xvf mujoco-*
 sudo mv mujoco-3.2.6 /opt/mujoco
 ```
 
+Alternatively, Mujoco can be built from source as explained under the section for MacOS below.
 
 ### macOS
 1. Install GLFW, a graphics library used by MuJoCo

--- a/README.md
+++ b/README.md
@@ -8,21 +8,28 @@ The derived reactors customize this base class for particular MuJoCo model files
 
 ### Linux (Ubuntu)
 
-Install GLFW, a graphics library used by MuJoCo
+1. Install GLFW, a graphics library used by MuJoCo
 
 ```sh
 apt install libglfw3-dev
 ```
 
+2. Download a prebuilt version of Mujoco v3.2.6 and install it to `/opt/mujoco`. The following works for x86, for aarch64, change the download path accordingly (https://github.com/google-deepmind/mujoco/releases/tag/3.2.6)
+```sh
+wget https://github.com/google-deepmind/mujoco/releases/download/3.2.6/mujoco-3.2.6-linux-x86_64.tar.gz
+tar xvf mujoco-*
+sudo mv mujoco-3.2.6 /opt/mujoco
+```
+
+
 ### macOS
-Install GLFW, a graphics library used by MuJoCo
+1. Install GLFW, a graphics library used by MuJoCo
 
 ```sh
 brew install glfw
 ```
 
-### All platforms
-Build Mujoco v3.2.6 from source and install to `/opt/mujoco`
+2. Build Mujoco v3.2.6 from source and install to `/opt/mujoco`
 
 ```sh
 git clone git@github.com:google-deepmind/mujoco.git -b 3.2.6
@@ -34,6 +41,8 @@ cmake --build .
 sudo cmake --install .
 
 ```
+
+### All platforms
 
 If mujoco is installed to a different location the
 [mujoco.cmake](src/include/mujoco.cmake) must be updated accordingly.

--- a/README.md
+++ b/README.md
@@ -32,7 +32,7 @@ brew install glfw
 2. Build Mujoco v3.2.6 from source and install to `/opt/mujoco`
 
 ```sh
-git clone git@github.com:google-deepmind/mujoco.git -b 3.2.6
+git clone https://github.com/google-deepmind/mujoco.git -b 3.2.6
 cd mujoco
 mkdir build
 cd build

